[2022-12-11 19:59:50,829] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world manual__2022-12-11T19:59:47.771434+00:00 [queued]>
[2022-12-11 19:59:50,847] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world manual__2022-12-11T19:59:47.771434+00:00 [queued]>
[2022-12-11 19:59:50,849] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:59:50,850] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-11 19:59:50,851] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:59:50,881] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): world> on 2022-12-11 19:59:47.771434+00:00
[2022-12-11 19:59:50,887] {standard_task_runner.py:52} INFO - Started process 5350 to run task
[2022-12-11 19:59:50,896] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'world', 'manual__2022-12-11T19:59:47.771434+00:00', '--job-id', '1297', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmp6tz83fkw', '--error-file', '/tmp/tmp5g068tj8']
[2022-12-11 19:59:50,899] {standard_task_runner.py:80} INFO - Job 1297: Subtask world
[2022-12-11 19:59:50,978] {task_command.py:369} INFO - Running <TaskInstance: first_dag.world manual__2022-12-11T19:59:47.771434+00:00 [running]> on host 7e39590a7807
[2022-12-11 19:59:51,096] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=world
AIRFLOW_CTX_EXECUTION_DATE=2022-12-11T19:59:47.771434+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-11T19:59:47.771434+00:00
[2022-12-11 19:59:51,099] {logging_mixin.py:115} INFO - Airflow
[2022-12-11 19:59:51,100] {python.py:173} INFO - Done. Returned value was: None
[2022-12-11 19:59:51,124] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=world, execution_date=20221211T195947, start_date=20221211T195950, end_date=20221211T195951
[2022-12-11 19:59:51,190] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-11 19:59:51,256] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
